assets/queries/terraform/aws/ecs_task_definition_network_mode_not_recommended/metadata.json

```diff
@@ -3,7 +3,7 @@
   "queryName": "ECS Task Definition Network Mode Not Recommended",
   "severity": "HIGH",
   "category": "Insecure Configurations",
-  "descriptionText": "Network_Mode should be 'awsvpc' in ecs_task_defenition. AWS VPCs provides the controls to facilitate a formal process for approving and testing all network connections and changes to the firewall and router configurations",
+  "descriptionText": "Network_Mode should be 'awsvpc' in ecs_task_definition. AWS VPCs provides the controls to facilitate a formal process for approving and testing all network connections and changes to the firewall and router configurations",
```

Comment:
The same typo (defenition) occurs in `assets/queries/openAPI/2.0/security_definitions_allows_password_flow/metadata.json` and `assets/queries/openAPI/2.0/security_definitions_using_basic_auth/metadata.json`. Can you also fix it, please? Thank you so much in advance!